src/Host/Client/Impl/Session/RSession.cs

```diff
@@ -571,8 +573,9 @@ internal sealed class RSession : IRSession, IRCallbacks {
         /// <param name="url"></param>
         /// <returns></returns>
         Task IRCallbacks.WebBrowser(string url) {
```

Comment:
>WebBrowser [](start = 25, length = 10)

AFAIK, this method is called for any URL, not just help. I don't think we should bother broker if it isn't a help URL.